test/e2e/performanceprofile/functests/utils/cgroup/v1/v1.go

```diff
@@ -0,0 +1,90 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/controller"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/runtime"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
+)
+
+type ControllersManager struct {
+	client    client.Client
+	k8sClient *kubernetes.Clientset
+}
+
+func NewManager(c client.Client, k8sClient *kubernetes.Clientset) *ControllersManager {
+	return &ControllersManager{client: c, k8sClient: k8sClient}
+}
+
+func (cm *ControllersManager) CpuSet(ctx context.Context, pod *corev1.Pod, containerName, runtimeType string) (*controller.CpuSet, error) {
+	cfg := &controller.CpuSet{}
+	cmd := []string{
+		"/bin/cat",
+		controller.CgroupMountPoint + "/cpuset/cpuset.cpus",
+	}
+	b, err := pods.ExecCommandOnPod(cm.k8sClient, pod, containerName, cmd)
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve cgroup config for pod. pod=%q, container=%q; %w", client.ObjectKeyFromObject(pod).String(), containerName, err)
+	}
+	output := strings.Split(string(b), "\r\n")
+	cfg.Cpus = output[0]
+	return cfg, nil
+}
+
+func (cm *ControllersManager) Cpu(ctx context.Context, pod *corev1.Pod, containerName, runtimeType string) (*controller.Cpu, error) {
+	cfg := &controller.Cpu{}
```

Comment:
We are passing runtimeType but we are not using this variable in Cpu function

Reply:
Yes I need to learn the details, i just passed the argument for now